locales/bn/bn.json

```diff
@@ -276,16 +276,16 @@
     "India": "ইন্ডিয়া",
     "Indonesia": "ফিজি",
     "Iran, Islamic Republic Of": "ফিনল্যাণ্ড",
-    "Iran, Islamic Republic of": "Iran, Islamic Republic of",
+    "Iran, Islamic Republic of": "ইরান (ইসলামি প্রজাতন্ত্র)",
```

Comment:
Why two different translations for Iran ?

Reply:
The [main source](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L278) has duplicate key, I'll delete duplicate

Reply:
No, my question is about the fact to have two different translations in Bengali for the same tag ; Of and of are normal.

Reply:
I got a lot of incorrect bengali translation, most of country names are incorrect, like 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L259) is `Honduras` but bengali translation is `England` (ইংল্যাণ্ড), 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L280) is `Iraq` and bengali translation is `France` (ফ্রান্স). 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L295) is `Kazakhstan` but bengali translation is `Greenland` (গ্রীনল্যাণ্ড),

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L311) is `Labanon` but bengali translation is `Hong Kong` (হং কং)

I didn't touch this files, [this](https://github.com/Laravel-Lang/lang/pull/1928/commits/8141e75118361f5ded730bdcdb1675775e902473#diff-592c253b8b0a994772dcdfd84e643773275bde1021ff21d24210d87947f1c831L278) is one of those.

Reply:
I'll send a new PR with all the country name